vmdb/app/models/mixins/password_mixin.rb

```diff
@@ -2,10 +2,15 @@ module PasswordMixin
   extend ActiveSupport::Concern
 
   included do
+    class_attribute :encrypted_column_list
```

Comment:
I still feel like this is weird having one class_attribute for the list, and then a separate method for the list again, `encrypted_columns` below.  There's gotta be a way to override it properly.